src/apis/Alert/index.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import View from '../../components/View';
+import Text from '../../components/Text';
+import TouchableOpacity from '../../components/Touchable/TouchableOpacity';
+
+let instance;
+
+let styles = {
+  background: {
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    right: 0,
+    bottom: 0,
+    backgroundColor: 'rgba(0,0,0,0.6)',
+    alignItems: 'center',
+    justifyContent: 'space-around'
+  },
+  container: {
+    overflow: 'hidden',
+    borderRadius: 10,
+    backgroundColor: '#f8f8f8'
+  },
+  textContainer: {
+    paddingTop: 20,
+    paddingHorizontal: 50,
+    alignItems: 'center',
+    justifyContent: 'space-around'
+  },
+  title: {
+    color: '#2b3137',
+    borderRadius: 2,
+    fontSize: 21,
+    fontWeight: 'bold'
+  },
+  message: {
+    color: '#2b3137',
+    marginTop: 4,
+    fontSize: 18,
+    maxWidth: 200,
+    textAlign: 'center'
+  },
+  buttonContainer: {
+    borderTopColor: '#dbdbdb',
+    flexDirection: 'row',
+    borderTopWidth: 1,
+    flex: 1,
+    marginTop: 20
+  },
+  buttonContainerMulti: {
+    flexDirection: 'column'
+  },
+  button: {
+    paddingVertical: 16,
+    alignItems: 'center',
+    justifyContent: 'space-around',
+    flex: 1
+  },
+  buttonSpacer: {
+    borderRightColor: '#dbdbdb',
+    borderRightWidth: 1
+  },
+  buttonSpacerMulti: {
+    borderBottomColor: '#dbdbdb',
+    borderBottomWidth: 1
+  },
+  buttonLabel: {
+    color: '#4b8ffe',
+    fontSize: 21
+  }
+};
+
+class AlertViewClass extends React.Component {
+
+  state={ visible: false };
+
+  onButtonPress = (onPress) => () => {
+    this.setState({ visible: false });
+    if (onPress) {
+      onPress();
+    }
+  };
+
+  render() {
+    const { visible, message, title, buttons } = this.state;
+    if (!visible) {
+      return <View />;
+    }
+    const isMultiButton = buttons.length > 2;
+    return (
+      <View style={styles.background}>
+        <View style={styles.container}>
+          <View style={styles.textContainer}>
+            <Text
+              style={styles.title}
+            >
+              {title}
+            </Text>
+            <Text
+              style={styles.message}
+            >
+              {message}
+            </Text>
+          </View>
+          <View style={[
+            styles.buttonContainer,
+            isMultiButton && styles.buttonContainerMulti
+          ]}>
+            {
+              buttons.map(
+                ({ text, onPress }, i) => (
+                  <TouchableOpacity
```

Comment:
In terms of accessibility, focus needs to move to the first button when the alert opens and return to the previously focused element after it closes